### PR TITLE
demistomock - convert docstring to google style

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/26399

## Description
Converting docstring style to google style as it looks better when rendering docs

### CODE REVIEW

File: Tests/demistomock/demistomock.py [hunk 1]
```diff
@@ -388,18 +388,19 @@ def params():
...
     return {}
 
 
 def args():
-    """
-    Retrieves a command / script arguments object
+    """Retrieves a command / script arguments object
+
+    Returns:
+      dict: Arguments object
```
Comment (ronykoz) [inline Tests/demistomock/demistomock.py:402]:
```suggestion
      dict. Arguments object
```
here and everywhere else

File: Tests/demistomock/demistomock.py [hunk 2]
```diff
@@ -408,37 +409,35 @@ def command():
...
 
-    :type msg: ``str``
-    :param msg: The message to be logged
+    Args:
+      msg (str): The message to be logged
 
-    :return: No data returned
-    :rtype: ``None``
+    Returns:
+      None: No data returned
```
Comment (ronykoz) [inline Tests/demistomock/demistomock.py:426]: no need to add that when no data returned

Comment (Itay4) [reply Tests/demistomock/demistomock.py:426]: need it so it will be clear that nothing is returned

File: Tests/demistomock/demistomock.py (hunk 1 above)
Comment (Itay4) [reply Tests/demistomock/demistomock.py:402]: both are ok actually

**Comment by glicht** [comment]
@Itay4 flake8 styling is failing

**Pull request merged**